coin/coin.py

```diff
@@ -39,12 +43,14 @@
             'name': 'Bitstamp',
             'instance': Bitstamp(config, indicator)
         },
-        {
-            'code': 'btce',
-            'name': 'BTC-E',
-            'instance': BtcE(config, indicator)
-        }
+
+        ## seized
+        #{
+        #    'code': 'btce',
+        #    'name': 'BTC-E',
+        #    'instance': BtcE(config, indicator)
+        #}
```

Comment:
No need to keep this around since it was seized, feel free to remove.

Reply:
hi Nil,

The Yaml allows you to configure the multiple instances when you launch startmany.py. Startmany parses the yaml and launches an instance for every entry. Quick and dirty, I know, better would be to rewrite coinprice-indicator so that one instance can show multiple tickers.

Reply:
I'm not talking about `startmany.yaml`, I'm talking about a `yaml` file that has 147,677 lines?

Reply:
whoops; that really doesn't belong there. Looks like it was generated by imagemagick when I was fooling around in that directory. Removed in latest commit

Reply:
Thanks. Can you also removed that code you commented out `## seized`

Reply:
removed!
@sebster hope you like it. Mind that this is just a quick and dirty and definitely not very optimal solution. Spawning different python processes is not the ideal way to go about this, but it works while I work on an OO solution